test/Core.Test/Services/CollectionServiceTests.cs

```diff
@@ -39,10 +44,15 @@ public async Task SaveAsync_DefaultIdWithUsers_CreatesCollectionInTheRepository(
         collection.Id = default;
         sutProvider.GetDependency<IOrganizationRepository>().GetByIdAsync(organization.Id).Returns(organization);
         var utcNow = DateTime.UtcNow;
+        // Grant the user manage access
+        var userList = users.ToList();
+        userList.First().Manage = true;
```

Comment:
question/thought: While not a problem, the list of `users` here will sometimes already have `Manage` set to true due to how the Autofixture creates objects with random values for properties. If we want the test to pass with the minimum requirements, we should probably forcefully set all but one of the users to `Manage=false`.

Alternatively, we could add a customization for `CollectionAccessSelection` to default all of the `Manager`, `Readonly`, `HidePasswords` fields to false and remove the level of randomness from the tests completely.

Up to you though, I don't think its super critical, but I think would strengthen the tests overall.

Reply:
Added a `CollectionAccessSelection` fixture to guarantee the state of the `Manage` boolean. This helped to clean up the tests as a nice side effect.